fix(content): Re-add events and missions to avoid breaking saves (#7399)
endless-sky/endless-sky: data/human/free worlds 1 start.txt

# To tell players to go to Longjump to resume the story if they were in the middle of one of the deleted missions.
mission "FW Pirates Patch"
	landing
	name "Return to <planet>"
	description "Return to <destination> to continue your work with the Free Worlds."

> 	destination Longjump

Reviewer: ```suggestion
	destination "Longjump"
```

	to offer
		or
			has "FW Pirates 3: offered"
			has "FW Pirates 3.1: offered"
			has "FW Pirates 3.2: offered"
			has "FW Pirates 3.2A: offered"
			has "FW Pirates 3.1B: offered"
			has "FW Pirates 3.2B: offered"
			has "FW Pirates 4: offered"
			has "FW Pirates 4B: offered"
			has "FW Pirates 4.1: offered"

> 		not "FW Pirates: Attack !: offered"

Reviewer: ```suggestion
		not "FW Senate Hearing: offered"
```

		not "FW Refinery 1: offered"
	on offer
		conversation
			`We've made some changes to the Free Worlds story. In order to continue, you should go to <destination>.`

> 				accept

Reviewer: ```suggestion
				accept

```

# To tell players to go to Longjump to resume the story if they were in the middle of one of the deleted missions.
mission "FW Pirates Patch"
	landing
	name "Return to <planet>"
	description "Return to <destination> to continue your work with the Free Worlds."
	source
		not planet "Longjump"
	destination "Longjump"
	to offer
		or
			has "FW Pirates 3: offered"
			has "FW Pirates 3.1: offered"
			has "FW Pirates 3.2: offered"
			has "FW Pirates 3.2A: offered"
			has "FW Pirates 3.1B: offered"
			has "FW Pirates 3.2B: offered"
			has "FW Pirates 4: offered"
			has "FW Pirates 4B: offered"
			has "FW Pirates 4.1: offered"
		not "FW Pirates: Attack 1: offered"
		not "FW Senate Hearing: offered"
		not "FW Refinery 1: offered"
	on offer
		conversation

> 			`We've made some changes to the Free Worlds story. In order to continue, you should go to <destination>.`

Reviewer: ```suggestion
			`When you updated your game, you were in the middle of a story arc in the Free Worlds campaign that has since been changed. In order to continue, you should go to <destination>.`
```
